### PR TITLE
Misc. UI tweaks

### LINKED ISSUE #5: Topmost collection page headings should be more distinct
Consider using a `bottom-border` or something similar.

### PR DESCRIPTION
**Pull request by tylersticka**
This PR includes a handful of UI tweaks, including:
- Fenced and joined pattern previews and code samples
- Fix for labels being too tall
- Removed `max-width` from main container (sorry @erikjung)
- Added proportional padding to main container at larger sizes
- Misc. tweaks for simplicity and consistent hierarchy
- Made the logo inherit the `currentColor` :neckbeard: 
## Before

screencapture-localhost-3000-patterns-typography-html-1461258632976
## After

screencapture-localhost-3000-patterns-typography-html-1461258606255

---

@erikjung @mrgerardorodriguez @saralohr @nicolemors 

Closes #5 #6

### CODE REVIEW

**Comment by gerardo-rodriguez** [comment]
Cool, LGTM! 👍

File: src/assets/drizzle/styles/components/item.css [hunk 1]
```diff
@@ -1,20 +1,5 @@
...
-
-  &:last-child {
-    border-bottom: none;
-  }
-}
-
-.Item-notes > :matches(:--heading) {
-  margin-bottom: calc(var(--space) / 2);
-  margin-top: calc(var(--space) * 1.5);
+  padding: calc(var(--space) * 1) 0;
```
Comment (erikjung) [inline src/assets/drizzle/styles/components/item.css]: Need the `calc()`?

Comment (tylersticka) [inline src/assets/drizzle/styles/components/item.css]: Ha, good catch!

**Comment by erikjung** [comment]
One nit, LGTM 👍

**Pull request merged**